src/lock_free/mwcas_descriptor.cpp

```diff
@@ -69,29 +70,43 @@ constexpr uint64_t kCntMask = (kMwCASFlag - 1UL) ^ (kPosMask | kAddrMask);
 /// @brief A bitmask with only the "MwCAS FLAG" and "descriptor address" portions set to 1.
 constexpr uint64_t kDescMask = kMwCASFlag | kAddrMask;
 
+}  // namespace
+
 /*##############################################################################
- * Local global variable
+ * Static utilities
  *############################################################################*/
 
-/// @brief A thread local descriptor for reuse.
-thread_local std::unique_ptr<MwCASDescriptor> _tls = nullptr;  // NOLINT
+void
+MwCASDescriptor::StartGC(  //
+    const size_t gc_interval,
+    const size_t gc_thread_num)
+{
+  _gc = std::make_unique<EpochBasedGC>(gc_interval, gc_thread_num, kMaxReusableDescriptors);
+}
 
-}  // namespace
+void
+MwCASDescriptor::StopGC()
+{
+  _gc.reset();
+}
+
+auto
+MwCASDescriptor::CreateEpochGuard()  //
+    -> ::dbgroup::thread::EpochGuard
+{
+  return _gc->CreateEpochGuard();
+}
 
 /*##############################################################################
  * Public APIs
  *############################################################################*/
 
 auto
-MwCASDescriptor::GetDescriptor()  //
-    -> MwCASDescriptor *
+MwCASDescriptor::GetDescriptor() -> MwCASDescriptor *
```

Comment:
余計な差分が入っているのでrevertで。

```suggestion
MwCASDescriptor::GetDescriptor()  //
    -> MwCASDescriptor *
```